client/vendor-dll.js

```diff
@@ -0,0 +1,13 @@
+// things we'd like to live in our vendor dll
+require( 'classnames' );
+require( 'debug' );
+require( 'i18n-calypso' );
+require( 'jed' );
+// lodash would go here, but we leave it out because we use tree shaking to pull parts of it
+require( 'moment' );
+require( 'page' );
+require( 'react-redux' );
+require( 'react' );
+require( 'redux' );
+require( 'store' );
+require( 'wpcom' );
```

Comment:
Has our vendor bundle not been tuned in a while? I'm only seeing:

```javascript
	webpackConfig.entry.vendor = [ 'react', 'store', 'page', 'wpcom', 'jed', 'debug' ];

```

in the current webpack.config.js